codes/rust/chapter_computational_complexity/worst_best_time_complexity.rs

```diff
@@ -0,0 +1,43 @@
+//   rand = "0.8.5"
```

Comment:
What's this

Reply:
Because of the use of the third library. You need to use `cargo single run` to compile and run a single file.
(rust does not have an official implementation of rand.)

Other examples can be done directly with `rustc`.

That's why there are some files I add how to run comment.

Reply:
I see. I meant `//   rand = "0.8.5"` should not be shown above the file header.

Reply:
Is there any more elegant approach to guide the compilation? Like CMakeList.txt for `C` and `C++`.

Reply:
If so, the carog.toml file needs to be added under each chapter, which is also available in the rust/ folder, for the purpose of to compile all executable files in all chapter at once